apps/api/src/app/invites/usecases/get-invite/get-invite.usecase.ts

```diff
@@ -32,6 +32,8 @@ export class GetInvite {
     const user = await this.userRepository.findById(invitedMember.invite._inviterId);
     if (!user) throw new NotFoundException('User not found');
 
+    const invitedUser = await this.userRepository.findByEmail(normalizeEmail(invitedMember.invite.email));
```

Comment:
Find the invited user by email and if exists set the `_userId`, this field is then used on the FE to show the "Sign In & Accept" or "Sign Up & Accept" forms.
Before this flag was set during the invite creation, which is wrong, because the user account can be created in between the invitation sent and the invitation accept (check the description for more).